Extensions/Lighting/lightruntimeobject.js

```diff
@@ -32,6 +35,14 @@ gdjs.LightRuntimeObject.prototype.getRadius = function () {
   return this._radius;
 };
 
+gdjs.LightRuntimeObject.prototype.getHeight = function () {
+  return 2 * this.getRadius();
+};
+
+gdjs.LightRuntimeObject.prototype.getWidth = function () {
+  return 2 * this.getRadius();
+};
```

Comment:
getHeight, getWidth are good :) 

I think we should consider, as mentioned in other comments, the center of the light as its x,y position. This would be especially useful with things like directional lights later, or just to be able to increase/reduce the radius of a light without having it moving ;) 
So in addition to these 2 methods, for the engine to understand that the top-left of the object is not at the position x,y, you can redefine the methods getDrawableX and getDrawableY:

```
gdjs.LightRuntimeObject.prototype.getDrawableX = function() {
    return this.x - this._radius;
};

gdjs.LightRuntimeObject.prototype.getDrawableY = function() {
    return this.y - this._radius;
};
```

This will be used by the engine for things like the draggable behavior, the culling (objects that are outside of the screen are not rendered), etc...